components/__tests__/NavBar.test.tsx

```
@@ -0,0 +1,91 @@
+describe('NavBar mandatory namespace functionality', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+    // Mock fetch for script loading
+    global.fetch = jest.fn().mockResolvedValue({ ok: false });
+  });
+
+  afterEach(() => {
+    jest.restoreAllMocks();
+    // Clean up fetch stub explicitly
+    // @ts-expect-error – fetch might be undefined in node
+    delete global.fetch;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)